syntax = "proto3";

package bookstore;

service BookService {
  rpc GetBooks (GetBooksRequest) returns (GetBooksResponse);
}

message GetBooksRequest {
}

message GetBooksResponse {
  repeated Book books = 1;
}

message Book {
  string id = 1;
  string title = 2;
  string author = 3;
}
